src/cbc_sdk/platform/asset_groups.py

```diff
@@ -200,6 +200,27 @@ def remove_members(self, members):
             self._cb.post_object(self._build_api_request_uri() + "/members",
                                  {"action": "REMOVE", "external_member_ids": member_ids})
 
+    def get_statistics(self):
+        """
+        For this group, return statistics about its group membership.
+
+        The statistics include how many of the group's members belong to other groups, and how many members
+        belong to groups without policy association.
+
+        See
+        `this page <https://developer.carbonblack.com/reference/carbon-black-cloud/platform/latest/asset-groups-api/#get-asset-group-stats>`  # noqa: E501 W505
```

Comment:
Minor tweak to the docs if possible.  The link doesn't display correctly.
<img width="722" alt="Screenshot 2023-12-20 at 3 37 23 PM" src="https://github.com/carbonblack/carbon-black-cloud-sdk-python/assets/55465092/2cd2a6c7-cd90-4e59-9959-3b00031e8311">